ecmascript/transforms/src/hygiene.rs

```diff
@@ -583,7 +583,10 @@ impl<'a> Fold for Hygiene<'a> {
 
     /// Invoked for `IdetifierRefrence` / `BindingIdentifier`
     fn fold_ident(&mut self, i: Ident) -> Ident {
-        if i.sym == js_word!("arguments") || i.sym == js_word!("undefined") {
+        if i.sym == js_word!("arguments")
+            || i.sym == js_word!("undefined")
+            || i.sym == js_word!("exports")
```

Comment:
`exports` is not a special keyword in javascript, so I'm not sure if preserving it from hygiene pass is correct.

Reply:
> exports is not a special keyword in javascript

hmmm, yes I agree, seems like I need to find another way to solve my problem.

Currently I got `exports1 is not defined` when I use `swc` with `lodash-es`

Reply:
If it's correct thing to do regarding the comment above, you can pass mark passed to resolver_with_mark to modules pass and apply those marks to exports and others.
However I'm not sure if it's correct way to go. I think lodash-es will break babel too if you apply common js module plugin to it.

Reply:
>  think lodash-es will break babel too if you apply common js module plugin to it

I will try later.

And I think maybe follow the `TypeScript` behavior would take less risk. And I will figure out how `TypeScript` handle this situation.